src/main/java/org/entando/plugins/pda/core/service/group/GroupService.java

```diff
@@ -0,0 +1,9 @@
+package org.entando.plugins.pda.core.service.group;
+
+import java.util.List;
+import org.entando.plugins.pda.core.engine.Connection;
+
+public interface GroupService {
+
+    List<String> list(Connection connection, String containerId, String processId);
```

Comment:
Should we use an id concatenated, same as we are using on Process Diagram endpoint?